.github/workflows/tests.yaml

```diff
@@ -100,12 +92,11 @@ jobs:
           mypy --ignore-missing-imports --strict src/
 
       - name: Test with black
```

Comment:
can't this whole step be removed? black is part of the self_check profile now:
https://github.com/sscpac/statick/blob/main/statick_tool/rsc/config.yaml#L122